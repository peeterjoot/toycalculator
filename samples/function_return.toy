FUNCTION bar ( ) : INT32
{
    RETURN 3;
};

INT32 v;
//PRINT "CALL foo";
v = CALL foo();
//PRINT v;
//PRINT "Back in main";
