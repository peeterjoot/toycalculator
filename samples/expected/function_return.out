CALL foo
Back in main
3
